assets/app/styles/_components.less

```diff
@@ -205,40 +215,75 @@
       .osc-object-highlight();
       .flex(@columns: 1 1 auto);
       max-width: 450px;
+      min-width: 90px;
+      background-size: 20px 20px;
       border-radius: @border-radius-md;
       border: 1px solid transparent;
-      // box-shadow: @osc-object-box-shadow; <- remove if it's not used in highlight
       padding: 10px;
       position: relative;
       margin: 0 10px 10px 0;
       text-align: center;
-      background-color: @pod-pending-bg;
+      .transition-duration(0.1s);
```

Comment:
don't bother with this for a .1s transition on color... just let it snap for now